libraries/AP_HAL/board/linux.h

```diff
@@ -227,6 +227,7 @@
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
 #define HAL_BARO_MS5611_NAME "ms5611"
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_BBBMINI
+#define HAL_AIRSPEED_MS4515DO_I2C_BUS 2
```

Comment:
This isn't actually used in the driver, so defining it has no effect.

@tridge Should we remove the macro from the driver or should we change code to use it?